frontend/src/components/Card.tsx

```diff
@@ -99,78 +98,76 @@ const Card = ({
-                          target="_blank"
-                          rel="noopener noreferrer"
-                          className="flex items-center justify-center gap-1"
-                        >
-                          <FontAwesomeIcon icon={item.icon as FontAwesomeIconProps['icon']} />
-                        </a>
-                      ))}
-                  </div>
-                )}
-              </div>
+          {(languages || topics || social) && (
+            <div className="mt-4 flex flex-col gap-3 md:flex-row">
+              {languages && (
+                <div className="flex flex-wrap gap-2">
+                  {languages.slice(0, visibleLanguages).map((topic, index) => (
+                    <TopicBadge
+                      key={topic || `language-${index}`}
+                      topic={topic}
+                      tooltipLabel={`This repository uses ${topic}`}
+                      type="language"
+                    />
+                  ))}
+                  {languages.length > 18 && (
+                    <button
+                      onClick={loadMoreLanguages}
+                      className="text-sm text-gray-600 dark:text-gray-300"
+                    >
+                      {toggleLanguages ? 'Show more' : 'Show less'}
+                    </button>
+                  )}
+                </div>
+              )}
+              {topics && (
+                <div className="flex flex-wrap gap-2">
+                  {topics.slice(0, visibleTopics).map((topic, index) => (
+                    <TopicBadge
+                      key={topic || `topic-${index}`}
+                      topic={topic}
+                      tooltipLabel={`This project is labeled as "${topic}"`}
+                      type="topic"
+                    />
+                  ))}
+                  {topics.length > 18 && (
+                    <button
+                      onClick={loadMoreTopics}
+                      className="text-sm text-gray-600 dark:text-gray-300"
+                    >
+                      {toggleTopics ? 'Show more' : 'Show less'}
+                    </button>
+                  )}
+                </div>
+              )}
+              {social && (
+                <div className="flex gap-2">
+                  {social.map((item) => (
+                    <a
+                      key={`${item.title}-${item.url}`}
+                      href={item.url}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="flex items-center justify-center text-sm"
+                    >
+                      <FontAwesomeIcon icon={item.icon as FontAwesomeIconProps['icon']} />
+                    </a>
+                  ))}
+                </div>
+              )}
             </div>
           )}
         </div>
-        <div className="w-38 content-end justify-items-center pr-6">
+        <div className="mt-4 w-full sm:mt-0 sm:w-auto">
```

Comment:
This now doesn't work with the regular screen size. The button should have a right padding that would match the rest of the card.

```suggestion
        <div className="mt-4 w-full sm:mt-0 sm:w-auto mr-6">
```

Reply:
Also, the button should always be at the top of the page, no matter what labels are there. 
For comparison, same card in current `main` state and with this PR:
![Screenshot 2024-12-21 at 6 35 57 PM](https://github.com/user-attachments/assets/e7f90ba5-6750-4f95-92fb-d7d03fc5124c)
![Screenshot 2024-12-21 at 6 36 19 PM](https://github.com/user-attachments/assets/bfb55141-0242-4a99-b998-1ee01fb1fa38)

Reply:
@kasya 
I am a bit confused about the positioning of the Contribute button. Could you please elaborate a little more on its placement?

![image](https://github.com/user-attachments/assets/0405dc50-6009-4c09-abeb-9ea9e8a3ed60)


![image](https://github.com/user-attachments/assets/62b4ebfd-ecac-4c25-a7b7-0cf55825037b)


![image](https://github.com/user-attachments/assets/4bc41ed8-5204-4da2-bde0-707d0757b001)